entity-service/src/main/kotlin/com/egm/stellio/entity/web/APIResponses.kt

```diff
@@ -16,5 +16,6 @@ data class BatchOperationResult(
 data class BatchEntityError(
     val entityId: URI,
     val error: MutableList<String>
-
 )
+
+const val ENTITY_NOT_FOUND_MESSAGE = "Entity Not Found"
```

Comment:
I think it can be moved to shared (in existing ApiUtils or new ApiResponses under shared/web) where we can add also other responses (i.e SUBSCRIPTION_NOT_FOUND_MESSAGE)

Reply:
Indeed, done, because it is also used by search service (but subscription specific messages will stay in subscription service).